eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -202,7 +202,7 @@ jobs:
           mkdir -p $(librariesDownloadDir)/bin/wasm/dotnet &&
           cp -r $(librariesDownloadDir)/BrowserWasm/staging/dotnet-latest/* $(librariesDownloadDir)/bin/wasm/dotnet &&
           cp -r $(librariesDownloadDir)/BrowserWasm/staging/built-nugets $(librariesDownloadDir)/bin/wasm &&
-          cp src/mono/wasm/test-main.js $(librariesDownloadDir)/bin/wasm/wasm-data/test-main.js &&
+          cp src/mono/browser/test-main.js $(librariesDownloadDir)/bin/wasm/wasm-data/test-main.js &&
```

Comment:
Did you tried to run the perf pipeline?

Reply:
https://github.com/search?q=repo%3Adotnet%2Fperformance%20test-main.js&type=code

Reply:
No, I have not, I don't know how.
I guess target folder stays the same, right ?

Reply:
`runtime-wasm-perf` pipeline should run automatically for this file.